src/modules/Elsa.Quartz/Elsa.Quartz.csproj

```diff
@@ -9,8 +9,6 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Quartz" />
-        <PackageReference Include="Quartz.Extensions.DependencyInjection" />
         <PackageReference Include="Quartz.Extensions.Hosting" />
```

Comment:
Relying on transitive behavior, Quartz 4 will eventually have these all built in o referencing just Quartz should be enough